### DISCUSSION TITLE
Chat response is fragmented/separated. How do I make it ignore whatever "new message" delimiter it sees?

### DISCUSSION THREAD

**Discussion by V3ntus** [LostRuins/koboldcpp#848] [Q&A] [answered]
![image](https://github.com/LostRuins/koboldcpp/assets/29584664/4ca0ed79-cbba-436a-a2f8-b6a3a835be67)

Ideally, I would like this to be one whole code block, not separated into multiple chat messages. How do I do that?

**Comment by LostRuins** [comment, marked as answer]
Are you generating it as one single response or multiple responses? Instruct mode should auto continue the previous response by default. What are you using for the start and end tags?

**Reply by V3ntus** [reply]
Yeah I believe that was the issue, I was using chat mode on an instruct model. Switching to instruct made the chats much more coherent
